### PR TITLE
chore: update prompt values color and heading

### PR DESCRIPTION
**Pull request by L-Joli**
updated:

list prompt

### CODE REVIEW

**Comment by laurawarr** [comment]
@L-Joli Do you think it would make sense to move the "Manage your <item>" above the current values? To show that the current values are scoped to the current prompt as well

**Review by L-Joli — COMMENTED** [review]
@laurawarr Updated!

**Review by laurawarr — APPROVED** [review]
Added one comment regarding the spacing, but otherwise looks good

File: src/commands/targeting/__snapshots__/update.test.ts.snap [hunk 1]
```diff
@@ -1,28 +1,39 @@
...
+
+
+🤖 Manage your filters
 🤖 Current Filters:
 ├─ All Users
 └─ AND User ID
    └─ is
       └─ not shrek
+
+
```
[the remark this answers is not in the document]
Comment (L-Joli) [reply src/commands/targeting/__snapshots__/update.test.ts.snap:25]: yup! have been thinking about how to better space the sections, ^and that makes sense!

**Pull request merged**